ossec_security/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# CHANGELOG - ossec-security
+
+<!-- towncrier release notes start -->
+
```

Comment:
You can browse some of the other changelogs in this repository to get a sense of various wordings, but at this stage the changelog should look something like this: https://github.com/DataDog/integrations-core/blob/master/amazon_eks/CHANGELOG.md

Reply:
@jhgilbert @ankitarajput-crest There has been some confusion about changelogs. The problem is we have 2 types:
1. Integrations that are shipped with the Agent
2. Integrations for which we only include tiles (assets and manifest)

This integration is type 1, so the changelog is correct for it.

For future reviews: if the changelog is incorrect, [this validation will fail](https://github.com/DataDog/integrations-core/actions/runs/10213533754/job/28259102223?pr=17938#step:29:16). If it passes, you're fine.